src/components/Button/button.css

```
@@ -0,0 +1,253 @@
+  justify-content: center;
+}
+
+.btn__spinner-icon {
+  animation: ui-spin 0.75s linear infinite;
+}
+
+/* ── Sizes ────────────────────────────────────────────────── */
+.btn[data-size="xs"] {
+  --_btn-h: var(--ui-space-6);
+  --_btn-px: var(--ui-space-2);
+  font-size: var(--ui-text-xs);
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)